test/test_collection.py

```diff
@@ -125,11 +142,11 @@ def test_delete_many(self):
         collection = client.db.products
         explain = ExplainCollection(collection)
         collection.delete_many({"status": "D"})
+        last_logger_payload = logger.cmd_payload
         res = explain.delete_many({"status": "D"})
         self.assertIn("queryPlanner", res)
         last_cmd_payload = explain.last_cmd_payload
-        for key in last_cmd_payload.keys():
-            assert last_cmd_payload[key] == logger.cmd_payload[key]
+        self._compare_command_dicts(last_cmd_payload, last_logger_payload)
 
     def test_watch(self):
```

Comment:
Let's skip this for now: https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures

Reply:
Done.